### ISSUE TITLE
Stage 1.2: Train root embeddings with tier-filtered vocabulary

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#618]
# Task

Train root embeddings using tier-filtered vocabulary (Fundamento + ReVo, exclude tier0 function words).

## Parent Epic

Part of **#616** (EPIC: Tier-Filtered Model Retraining)

## Model Architecture

**Simple embedding table** (not compositional):
```python
class RootEmbeddings(nn.Module):
    self.embeddings = nn.Embedding(vocab_size, embedding_dim)  # ~9,800 × 64D
```

## Training Configuration

- **Vocabulary**: ~9,800 roots (tier 1a+1b+2)
- **Embedding dim**: 64D
- **Training objective**: Co-occurrence prediction (contrastive learning)
- **Epochs**: 100 (early stopping patience=15)
- **Batch size**: 128
- **Learning rate**: 0.001

## Training Data

From **#617**: `data/training/ekzercaro_tier_filtered.jsonl`

## Output

File: `models/root_embeddings/best_model.pt`

Checkpoint contains:
- `model_state_dict`: Embedding weights (9,800 × 64)
- `root_to_idx`: Vocabulary mapping
- `embedding_dim`: 64
- `correlation`: Best similarity correlation score

## Acceptance Criteria

- [ ] Vocabulary size: ~9,800 roots (±100)
- [ ] No tier0 function words in vocabulary (validation required)
- [ ] Training converges (correlation improves)
- [ ] Model saved with all checkpoint fields
- [ ] Training logs show no errors

## Command

```bash
./scripts/train_roots.sh --fresh
```

## Dependencies

- Blocked by: #617 (Generate training data)
- Blocks: #619 (Test quality)

**Comment by marctjones** [comment]
Closing as part of a 2026-09-08 backlog cleanup (see PLAN.md, DESIGN.md, CLAUDE.md's merge gate).

This issue predates the merge gate (adopted 2026-07-13) and the AST-native orchestrator rewrite. It describes work against the pre-orchestrator RAG pipeline or the learned model stack (M1/M2 embeddings, compositional embeddings, entity classifiers, neural rerankers) that was pruned from the repo -- klareco/embeddings/, klareco/models/, and klareco/summarization/ no longer exist. It has no milestone and was never benchmarked, matching exactly the failure mode CLAUDE.md documents (25 capability issues filed 2026-05-26, 8 implemented, none benchmarked).

Labeled archived-2026-09 so this batch is recoverable: gh issue list --state closed --label archived-2026-09. If this capability is still wanted, re-file it against the current architecture naming the benchmark metric and test the merge gate requires.
